docs/content/get-started/developers.mdx

```diff
@@ -229,46 +229,92 @@ Customize the design system with settings and custom code.
 
 The USWDS [Settings](https://designsystem.digital.gov/documentation/settings/) page describes each of the settings available in the settings files, as well as the values the settings accept. Most settings accept design tokens. Visit the [Foundations](/foundations) section of our website for more information on the available tokens for color, spacing units, font size, and more.
 
-## CDN
+## Alternative Methods
 
-### NCIDS minimum
+### Content Delivery Network
 
-```html
-<link
-	href="http://designsystem.cancer.gov/cdn/ncids-v3.4.0/ncids-minimum.min.css "
-	rel="stylesheet"
-	crossorigin="anonymous"
-/>
-<script
-	src="http://designsystem.cancer.gov/cdn/ncids-v3.4.0/ncids-minimum.min.js"
-	crossorigin="anonymous"
-></script>
-```
+While we encurage using the NCIDS via npm as it gives you more flexibility, it is possible to use it without installing, compiling, or customizing the code. To do so, directly include the stylesheet and JavaScript in your HTML. Then update your HTML to utilize the NCIDS CSS classes.
+
+We provide three options via a Content Delivery Network (CDN). Choose the one that works best for your use case.
 
-### NCIDS without utilities
+import useSiteMetadata from '../../src/hooks/use-site-metadata';
+import { withPrefix } from 'gatsby';
+import PropTypes from 'prop-types';
 
-```html
-<link
-	href="http://designsystem.cancer.gov/cdn/ncids-v3.4.0/ncids.min.css "
+export const CDNCode = ({ cssFile, jsFile }) => {
+	CDNCode.propTypes = {
+		cssFile: PropTypes.string.isRequired,
+		jsFile: PropTypes.string.isRequired,
+	};
+	const siteUrl = useSiteMetadata()?.siteUrl;
+	const v = useSiteMetadata()?.versionInfo?.ncidsVersion ?? 'ERROR';
+	const cssPath = withPrefix(`/ncids-${v}/ncids-css/${cssFile}`);
+	const jsPath = withPrefix(`/ncids-${v}/ncids-js/${jsFile}`);
+	const snippet = `<link
+	href="${siteUrl}/cdn${cssPath}"
```

Comment:
We stick `cdn` here in between the site URL and the paths we calculated so we get the correct link to the CDN folder. If we put the `cdn` inside the `withPrefix` call then we'd end up with something like `pr-X/cdn` instead of `cdn/pr-X`.